{
    "CVE_data_meta": {
        "ASSIGNER": "cve@mitre.org",
        "ID": "CVE-2010-1666",
        "STATE": "PUBLIC",
        "UPDATED": "2020-06-15"
    },
    "cna-container": {
        "affected": {
            "vendors": [
                {
                    "products": [
                        {
                            "versions": [
                                {
                                    "version_value": "n/a"
                                }
                            ]
                        }
                    ],
                    "vendor_name": "n/a"
                }
            ]
        },
        "descriptions": [
            {
                "lang": "eng",
                "value": "Buffer overflow in Dan Pascu python-cjson 1.0.5, when UCS-4 encoding is enabled, allows context-dependent attackers to cause a denial of service (application crash) or possibly have unspecified other impact via vectors involving crafted Unicode input to the cjson.encode function."
            }
        ],
        "problemtypes": {
            "descriptions": [
                [
                    {
                        "description": "n/a",
                        "lang": "eng",
                        "type": "text"
                    }
                ]
            ]
        },
        "references": [
            {
                "name": "ADV-2010-1774",
                "refsource": "VUPEN",
                "url": "http://www.vupen.com/english/advisories/2010/1774"
            },
            {
                "name": "DSA-2068",
                "refsource": "DEBIAN",
                "url": "http://www.debian.org/security/2010/dsa-2068"
            },
            {
                "name": "40335",
                "refsource": "SECUNIA",
                "url": "http://secunia.com/advisories/40335"
            },
            {
                "name": "40500",
                "refsource": "SECUNIA",
                "url": "http://secunia.com/advisories/40500"
            },
            {
                "name": "https://bugs.launchpad.net/ubuntu/+source/python-cjson/+bug/585274",
                "refsource": "CONFIRM",
                "url": "https://bugs.launchpad.net/ubuntu/+source/python-cjson/+bug/585274"
            }
        ]
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "5.0"
}